index.html

```diff
@@ -329,6 +329,37 @@ <h4>Click to see bio</h4>
             </div>
           </div>
         </div>
+        <div class="row" style="margin-top: 70px">
+          <div class="col-lg-7" style="display: flex; align-items: center;">
+            <div class="speaker-bio d-none d-sm-block">
+              <b>Cadran Cowansage</b> is the CEO and Founder of <a target="_blank" href="https://elpha.com/" class="black-link"><b>Elpha</b></a>, an online community dedicated to accelerating the careers of women in the tech industry. The Elpha community is tens of thousands of members strong and includes women at all levels in software engineering, product, sales, marketing, design, and more. Elpha is part of Y Combinator’s Summer 2019 Batch and has raised over $1 million in seed funding. Before founding Elpha, Cadran was a Lead Software Engineer at Y Combinator, and before that a Senior Software Engineer at MongoDB. She holds a Masters in Biomedical Informatics from Columbia University and an undergraduate degree in Computer Science and Biology from Colby College.
+            </div>
+            <div class="d-block d-sm-none speaker-bio">
+              <div class="accordion">
+                  <h4>Click to see bio</h4>
+                  <div class="faq-toggle-icon"></div>
+              </div>
+              <div class="panel">
```

Comment:
hmm yeah i think it still looks awk, lets keep the speaker pic on the left and the bio on the right

Reply:
LOL i thought i changed it before oops 🐸